src/test/cucumber/de/filefighter/rest/ViewFolderContents.feature

```diff
@@ -0,0 +1,22 @@
+Feature: View Folder
+  All users want to see the content of folders and navigate in them, so they can see and interact with their uploaded and shared files.
+
+Scenario: Successful interaction
+  Given credential are valid
```

Comment:
I think we should create a authorization step too or maybe first, so we can use this step. 
Or Better you should do it this way: 
`Given Database contains User with name Kevin, refreshToken token username..... pw....`  
`When the user (same as above) requests to see the content of folder foo/bar with the accessToken accessToken` etc...
I mean by that that the `given credentials do not exist` step is not really good here.
Also i think that you can create a precondition for each scenario like
`user db is empty`  
`dummy user exists in db`  
and after that checking the scenarios

Reply:
But auth should not be important for this UC, refreshToken token username..... pw would be overhead

Reply:
the step `given credentials are valid` make sense tho because that's not implementable. you have to specify that there is a access token for the specific user that is viable when the test is run, and that the user consumes the endpoints with that token. basically you have to specify all the information that is necessary to run the endpoint function. And of course the surroundings. You could argue that it doesn't matter whether the db is empty or not but i think it is best practise to do so.